lib/lotus/generators/database_config.rb

```diff
@@ -0,0 +1,88 @@
+module Lotus
+  module Generators
+    class DatabaseConfig
+      SUPPORTED_ENGINES = {
+        'mysql'      => { type: :sql,         mri: 'mysql',   jruby: 'jdbc-mysql'    },
```

Comment:
I don't think we should support the old `mysql` driver anymore, can you please change it to `mysql2` for MRI?